pkg/controller/plan/adapter/ova/ovfparser.go

```diff
@@ -6,9 +6,10 @@ import (
 )
 
 type OvaVmconfig struct {
-	XMLName xml.Name `xml:"domain"`
-	Name    string   `xml:"name"`
-	OS      OS       `xml:"os"`
+	XMLName  xml.Name `xml:"domain"`
+	Name     string   `xml:"name"`
+	OS       OS       `xml:"os"`
+	Metadata Metadata `xml:"metadata"`
```

Comment:
IIRC we talked briefly about adding this to OVA as well,  so it make sense to update it for both providers.

Reply:
it would make sense to do that also for OVA, but the description of the PR talks specifically about vSphere ("When migrating from vSphere...") and we need to minimize the changes in 2.6 - I'm ok with doing the changes for OVA in a separate PR

Reply:
yes, but the parser sits here, unless i wish to pull it out entirely and make a bigger change - i used it where it is now.

Reply:
ah I see, it's placed incorrectly in the `ova` package even though it's now used also for vsphere providers
it needs to move elsewhere but let's do that separately